auto_round/inference/backend.py

```diff
@@ -172,6 +172,22 @@ def feature_multiply_checker_group_size(
     requirements=["auto-round>=0.5.1"],
 )
 
+# FP8 static quant
+# Weight: FP8, per-channel, may be extended to per-tensor in future
+# Activation: FP8, per-tensor
+
+BackendInfos["auto_round:torch_fp8_static"] = BackendInfo(
+    device=["xpu", "cuda", "cpu"],
+    packing_format="",
+    sym=[True],
+    dtype=["float32", "float16", "bfloat16"],
+    bits=[8],
+    priority=0,
```

Comment:
Add checkers to differentiate between w8a8-FP8 dynamic, w8a8-int, w8a4, etc.

Reply:
1 no, that's not the logic.  If we have another backend that could support W8A8 int8, it may match the fp8 backend
2 follow the backend style could avoid lots of tricky code like if else

So please follow our backend coding stype

Reply:
another way is you could add a checker  with function  just like feature checks

Reply:
updated